Library/Homebrew/dev-cmd/typecheck.rb

```diff
@@ -16,6 +18,8 @@ def typecheck_args
              description: "Silence all non-critical errors."
       switch "--update-definitions",
              description: "Update Tapioca gem definitions of recently bumped gems"
+      switch "--prune-files-list",
+             description: "Remove deleted filepaths from #{SORBET_FILES_YAML}"
```

Comment:
Thoughts on making this run unconditionally?

Reply:
Or: as part of `update-definitions`?

Reply:
It's a distinct task, so I feel it should be separate.

I'm already slightly uncomfortable with it being in a GitHub Action named "Tapioca" because it's core Sorbet not Tapioca. But not uncomfortable enough to go to the effort of fixing it as that GitHub Action was difficult enough the first time.

Reply:
We can rename the Git workflow/branches/etc. to `typecheck`?

It's a separate task but if it's only ever run with `--update-definitions` (and we're the only users of these) it feels like maybe even a `--ci` flag or similar might be valuable to group them all?

Reply:
I'd be very surprised if anyone else were using those options of `brew typecheck` at the moment - or even the base command. That sounds good - I'll do that a bit later today.